src/TestApp/shared/App.xaml.cs

```diff
@@ -168,7 +168,7 @@ private static void InitializeLogging()
 				builder.AddFilter("Uno.UI.RuntimeTests.HotReload", LogLevel.Debug);
 
 				// Hot reload testing
-				builder.AddFilter("Uno.UI.RemoteControl", LogLevel.Debug);
+				builder.AddFilter("Uno.UI.RemoteControl", LogLevel.Trace);
```

Comment:
Are you sure to keep this on by default? It's probably too intense. Debug I'd find somehow interesting, if not configurable another way.

Reply:
Yes it was on purpose: this is only for the "test app" and we have almost no logs if in debug ... I would keep it like that and reduce it later if annoying.